packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -16,6 +16,14 @@ import 'scroll_simulation.dart';
 
 export 'package:flutter/physics.dart' show ScrollSpringSimulation, Simulation, Tolerance;
 
+/// The rate at which scroll momentum will be decelerated.
+enum ScrollDecelerationRate {
+  /// The standard deceleration expected on touchscreen apps.
+  normal,
+  /// Faster deceleration aligned with desktop software conventions.
```

Comment:
Is desktop generally faster?
This enum sounds vague, what is it meant to convey? What happens if later there is a faster one, how would we describe it?

Reply:
I just copied it from these: https://developer.apple.com/documentation/uikit/uiscrollviewdecelerationrate.
Do you think it's better to just use mobile / desktop?

Reply:
I am not sure. My main concern here is future changes and what it means to users. @goderbauer do you have any thoughts on this? 

Since this is specific to Apple, I don't know that it quite translates here in the abstract. Is Windows faster than touchscreens? Is Linux? What about desktop platforms that have touchscreens?

If we say that desktop is faster and touch screens are slower, we should be sure in order to avoid confusion about this behavior.

If this enum is specific to only the mac and ios scrolling behaviors, then maybe we can make it more specific that way.

Reply:
I found that Windows and Linux have no different behaviour on scroll deceleration for trackpad v.s. touchscreen. But that's a sample size of one device. Do you think eventually `ClampingScrollPhysics` will have a desktop variant as well?

In my opinion desktop should always have greater deceleration / less momentum, since the input device is more precise and is not 1-to-1 with screen coordinates.

Calling it `ScrollDecelerationRate` is also kind of a lie since it affects other things as well (velocity tracker, overscrolling), but with the amount of scroll abstractions already in Flutter it is hard to find a good name to cover that.

Reply:
> In my opinion desktop should always have greater deceleration / less momentum, since the input device is more precise and is not 1-to-1 with screen coordinates.

That makes so much sense. Perhaps including that in the docs, especially about precision input devices, will make it a bit clearer. It made it totally clear to me. :)